docs/advanced/input_files/input-main.md

```diff
@@ -587,37 +587,37 @@ These variables are used to control parameters related to input files.
 ### stru_file
 
 - **Type**: String
-- **Description**: This parameter specifies the name of structure file which contains various information about atom species, including pseudopotential files, local orbitals files, cell information, atom positions, and whether atoms should be allowed to move.
+- **Description**: Specifies the name of the structure file containing various information about atom species, including pseudopotential files, local orbitals files, cell information, atom positions, and whether atoms should be allowed to move.
 - **Default**: STRU
 
 ### kpoint_file
 
 - **Type**: String
-- **Description**: This parameter specifies the name of k-points file. Note that if you use atomic orbitals as basis, and you only use gamma point, you don't need to have k-point file in your directory, ABACUS will automatically generate `KPT` file. Otherwise, if you use more than one k-point, please do remember the algorithm in ABACUS is different for gamma only and various k-point dependent simulations. So first you should turn off the k-point algorithm by set `gamma_only = 0` in `INPUT` and then you should setup your own k-points file.
+- **Description**: Specifies the name of the k-points file. Note that if you use atomic orbitals as basis and only use the gamma point, you don't need a k-point file in your directory. ABACUS will automatically generate a `KPT` file. However, if you use more than one k-point, please remember that the algorithm in ABACUS is different for gamma-only and various k-point dependent simulations. First, turn off the k-point algorithm by setting `gamma_only = 0` in `INPUT`, and then set up your own k-points file.
```

Comment:
```suggestion
- **Description**: the name of the k-points file
- **Note**: In atomic orbitals basis and gamma only calculation, the `KPT` file is unnecessary, because a `KPT` file will be generated automatically. However, if more than one k-point are required, a explicit `KPT` file becomes mandatory.